test/Crypto/Test/Hash/Multi.hs

```diff
@@ -29,7 +29,7 @@ tests = testGroup "Multihash"
                 example BaseN.Base16 C.SHA1 "multihash"
                     "111488c2f11fb2ce392acb5b2986e640211c4690073e"
             , testCase "Base58" $
-                example BaseN.Base58 C.SHA1 "multihash"
+                example BaseN.Base58btc C.SHA1 "multihash"
```

Comment:
Not sure we need this test module anymore (as per your FIXME)

Reply:
Gonna remove them -- opened https://github.com/oscoin/ipfs/issues/51 to not forget.